module/Decision/Module.php

```diff
@@ -145,14 +155,21 @@ public function getServiceConfig()
 
                     $acl->allow('user', 'authorization', ['create', 'view_own']);
 
+                    // users are allowed to use the filebrowser
+                    $acl->allow('user', 'files', 'browse');
+
                     return $acl;
                 },
                 // fake 'alias' for entity manager, because doctrine uses an abstract factory
                 // and aliases don't work with abstract factories
                 'decision_doctrine_em' => function ($sm) {
                     return $sm->get('doctrine.entitymanager.orm_default');
                 }
-            ]
+            ],
+            /*
+             * Regex pattern matching filenames viewable in the browser
+             */
+            'filebrowser_valid_file' => '[^?*:;{}\\\]*'
```

Comment:
I wasn't sure if this is the appropriate location for a config constant that should be used in both `Module.php` and `module.config.php` of the same module.